### PR TITLE
feat: Disable Login and SignUp Button until inputs entered

### LINKED ISSUE #594: Disable signup/login button until and unless user fills necessary fields
**What is the feature about?**
Disable login and signup buttons until and unless user fills the necessary fields (Email and Password) in case of login and (Email, Password, Confirm Password) in case of Sign-Up, and enable it back as soon as these fields are filled.

**ScreenShots of the Issue**
whatsapp image 2018-09-29 at 10 14 59 am 1 whatsapp image 2018-09-29 at 10 14 59 am

**Would you like to work on this issue?**
Yes

### PR DESCRIPTION
**Pull request by kshitijsuri90**
Fixes #594 

Changes:
Activity instance variable added with a method to change UI of the 
button.

Screenshots for the change:
login_disable

@liveHarshit @iamareebjamal @nikit19 Do review

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt [hunk 1]
```diff
@@ -140,4 +161,17 @@ class SignUpFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun setSignUpState() {
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt]: This should also check if the password is same then only allow the user to click

Comment (kshitijsuri90) [reply app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt]: Done @nikit19

**Comment by kshitijsuri90** [comment]
@nikit19 @liveHarshit @iamareebjamal Do review

**Comment by kshitijsuri90** [comment]
Yes the login button is disabled when no input is entered but there is no UI change showing that it is disabled. Eventbrite also follows the same

**Comment by liveHarshit** [comment]
Okay then, do changes in UI and upload GIF for signup also.

**Comment by kshitijsuri90** [comment]
The main gif already has sign up UI and changes @liveHarshit .

**Pull request closed without merging**